doc/nrf/config_and_build/zephyr_samples_sysbuild.rst

```diff
@@ -0,0 +1,41 @@
+| :kconfig:option:`SB_CONFIG_NETCORE_802154_RPMSG`        | ``802154_rpmsg``        | ✕         | ✓        |                                                                                                                       |
++---------------------------------------------------------+-------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` | ``multiprotocol_rpmsg`` | ✓         | ✓        |                                                                                                                       |
++---------------------------------------------------------+-------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO`           | ``ipc_radio``           | ✓         | ✓        | Requires additional configuration: :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC` enables HCI serialization |
+|                                                         |                         |           |          | for Bluetooth, :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC` enables nRF RPC serialization for Bluetooth, or   |
+|                                                         |                         |           |          | :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_IEEE802154` enables spinel serialization for IEEE 802.15.4.              |
++---------------------------------------------------------+-------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_NETCORE_NONE`                | No image                | -         | -        |                                                                                                                       |
++---------------------------------------------------------+-------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+
+The default for thing53 is the ``empty_net_core`` sample application, for other nRF53 devices the default is no image.
+When configuring an application, for example :zephyr_file:`zephyr/samples/bluetooth/peripheral_hr`, it must be configured with a supported network core image to work, for basic Bluetooth samples ``hci_ipc``, ``multiprotocol_rpmsg`` or ``ipc_radio`` can be used.
+The following shows how to build a sample with the ``hci_ipc`` network image selected:
```

Comment:
```suggestion
When configuring an application, such as :zephyr_file:`zephyr/samples/bluetooth/peripheral_hr`, you must configure it with a supported network core image to ensure proper functionality.
For basic Bluetooth samples, you can use `hci_ipc`, `multiprotocol_rpmsg`, or `ipc_radio`.

The following instructions demonstrate how to build a sample with the `hci_ipc` network image selected:
```
Not sure if what you are trying to say here is that if you need a supported network core image to combine with your app running on the app core, you can use one of the 3 basic bluetooth samples listed above.